control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -681,6 +707,9 @@ impl From<VolumeOperation> for models::volume_spec_operation::Operation {
             VolumeOperation::DestroySnapshot(_) => {
                 models::volume_spec_operation::Operation::DestroySnapshot
             }
+            VolumeOperation::SetVolumeProperties(_) => {
+                todo!()
```

Comment:
need to update models here, otherwise we panic...

Reply:
This will be filled once, openapi changes are in placed. Planned to put it in another PR.
Draft PR https://github.com/openebs/mayastor-control-plane/pull/752 is there for reference.